geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/RenderingGrid.scala

```diff
@@ -218,6 +218,7 @@ class RenderingGrid(env: Envelope, xSize: Int, ySize: Int) extends LazyLogging {
         i += 1
       }
     }
+    count += 1
```

Comment:
How did you reach the conclusion that we needed to update this line?

The counts for the RenderingGrid seem to be very loose/complex.

Reply:
yeah... right now they're not actually used though. i realized it was wrong b/c i had an initial implementation that did rely on the counts to track the features and it was failing due to the missing +1 here. i figured i should fix it regardless.
The original thought was to count the number of pixels that got rendered, but i think that is kind of irrelevant anyway.